frontend/src/components/rota/rota.module.scss

```diff
@@ -1,5 +1,16 @@
 .rota {
   grid-area: main;
-  width: 100%;
-  height: 100%;
+  display: grid;
+  height: 100vh;
+  margin: 0 1em;
+  grid-template:
+    'header' min-content
+    'calendar' 1fr;
+  h1 {
+    grid-area: header;
+  }
+}
+
+.calendarContainer {
```

Comment:
> I changed the colour of 'today' header to yellow, but I couldn't do this in the rota's scss file, so I had to put it in main.css.
> My guess is that scss is compiled before fullcalendar.io classes are generated, so the styling is not applied to them. I have checked that the styling is not applied at all, it's not overwritten.
> There may be a way around this by e.g. playing with webpack's settings. Suggestions welcome.

would you have a similar result if instead you set the styling here with the global tag?

so something like:
``` 
.calendarContainer {
  grid-area: calendar;
  
  & :global(.fc-today) {
    background-color: yellow;
  }
}
```

Havent tried myself but I think that should make any child class of `calendarContianer` with the name `fc-today` get that styling.

Reply:
That worked, thanks!